bundles/org.openhab.binding.omnikinverter/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,54 @@
+<thing:thing-descriptions bindingId="omnikinverter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="omnik">
+		<label>OmnikInverter Binding Thing</label>
+		<description>Thing for OmnikInverter Binding</description>
+
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="energyToday" typeId="energyToday" />
+			<channel id="energyTotal" typeId="energyTotal" />
+		</channels>
+
+		<config-description>
+			<parameter name="omnikHostname" type="text" required="true">
```

Comment:
It's not really necessary to prefix all the parameters with `omnik` as there used only in the context of the omnik.